waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -6,33 +6,40 @@ module Wasp.Generator.DbGenerator
     dbRootDirInProjectRootDir,
     dbMigrationsDirInDbRootDir,
     dbSchemaFileInProjectRootDir,
+    writeDbSchemaChecksumToFile,
+    afterWriteChecks,
   )
 where
 
 import Control.Monad (when)
 import Data.Aeson (object, (.=))
 import Data.Maybe (fromMaybe, isNothing, maybeToList)
-import StrongPath (Abs, Dir, File', Path', Rel, reldir, relfile, (</>))
+import StrongPath (Abs, Dir, File, File', Path', Rel, reldir, relfile, (</>))
 import qualified StrongPath as SP
-import System.Directory (doesDirectoryExist, removeDirectoryRecursive)
-import Wasp.AppSpec (AppSpec)
+import System.Directory (doesDirectoryExist, doesFileExist, removeDirectoryRecursive)
+import Wasp.AppSpec (AppSpec, getEntities)
 import qualified Wasp.AppSpec as AS
 import qualified Wasp.AppSpec.App as AS.App
 import qualified Wasp.AppSpec.App.Db as AS.Db
 import qualified Wasp.AppSpec.Entity as AS.Entity
 import Wasp.Common (DbMigrationsDir)
 import Wasp.Generator.Common (ProjectRootDir)
 import Wasp.Generator.FileDraft (FileDraft, createCopyDirFileDraft, createTemplateFileDraft)
-import Wasp.Generator.Monad (Generator, GeneratorError (..), logAndThrowGeneratorError)
+import Wasp.Generator.Monad (Generator, GeneratorError (..), GeneratorWarning (GenericGeneratorWarning), logAndThrowGeneratorError)
 import Wasp.Generator.Templates (TemplatesDir)
 import qualified Wasp.Psl.Ast.Model as Psl.Ast.Model
 import qualified Wasp.Psl.Generator.Model as Psl.Generator.Model
-import Wasp.Util ((<:>))
+import Wasp.Util (Hex (Hex), checksumFromFilePath, (<:>))
 
 data DbRootDir
 
 data DbTemplatesDir
 
+-- | This file represents the checksum of schema.prisma
+-- at the point at which db migrate-dev is run. It is used
```

Comment:
```suggestion
-- at the point at which db migrate-dev was last run. It is used
```

Reply:
While I am writing this, I wonder if we should encode this in name? Should it maybe be called `DbSchemaChecksumOnLastMigrateFile`? Or am I just pushing it too far with this? The reasoning is that when I see `DbSchemaChecksumFile` I don't really know what it is. But maybe that is ok not sure. Then we will need longer name everywhere else which is PITA. Hm.

Reply:
I can update the comment and see about how the name change looks/feels in use 👍🏻